src/hotspot/share/runtime/flags/jvmFlagConstraintsCompiler.cpp

```diff
@@ -228,6 +228,15 @@ JVMFlag::Error CodeEntryAlignmentConstraintFunc(intx value, bool verbose) {
       return JVMFlag::VIOLATES_CONSTRAINT;
   }
 
+  if ((uintx)CodeEntryAlignment > CodeCacheSegmentSize) {
+    JVMFlag::printError(verbose,
+                        "CodeEntryAlignment  (" INTX_FORMAT ") must be "
```

Comment:
```suggestion
                        "CodeEntryAlignment (" INTX_FORMAT ") must be "
```

Reply:
Fixed.

Thanks @vnkozlov and @TobiHartmann for your review.